source/index.js

```diff
@@ -1,9 +1,14 @@
 import compose from './compose';
-import init from './init';
-import isStamp from './is-stamp';
-import isDescriptor from './is-descriptor';
-import isComposable from './is-composable';
+const isFunction = obj => typeof obj === 'function';
+const isObject = obj => !!obj && (typeof obj === 'function' || typeof obj === 'object');
 
+const isDescriptor = obj => isObject(obj);
```

Comment:
I think the descriptor could be `{}` and still be valid so I'm not sure what we would check for.